aws/logs_monitoring/README.md

```diff
@@ -344,6 +352,9 @@ To test different patterns against your logs, turn on [debug logs](#troubleshoot
 `DdUsePrivateLink`
 : Set to true to enable sending logs and metrics via AWS PrivateLink. See https://dtdg.co/private-link.
 
+`DdHttpProxyURL`
+: For web proxy support, sets the HTTP_PROXY and HTTPS_PROXY values. Don't use this in combination with AWS Private Link. Make sure to also set DdSkipSslValidation to true, and
+
```

Comment:
I think this sentence is incomplete (ends with "and")
That said, I think I would rephrase this. What are the `HTTP_PROXY` and `HTTPS_PROXY` values? Why do they matter? It might be better to back up a bit and explain a bit what this setting is for at a higher level.

Reply:
I added a little bit of information, but it's kind of a big topic to cover. It should be clear to someone who has set up web proxy, what those variables are for. I think with this, plus the steps in the proxy in section, it should be enough to cover how to use it.